docs/dsp-tools-create-ontologies.md

```diff
@@ -797,13 +797,13 @@ The following base resources can be used as super-resource:
 
 - `Resource`: A generic resource representing an item from the real world. This is the most general case, to be 
 used in all cases when your resource is none of the special cases below.
-- `StillImageRepresentation`: A resource representing an image
-- `TextRepresentation`: A resource representing a text
+- `ArchiveRepresentation`: A resource representing an archive file (e.g. ZIP)
 - `AudioRepresentation`: A resource representing an audio file
 - `DDDRepresentation`: A resource representing a 3-D representation (not yet implemented)
 - `DocumentRepresentation`: A resource representing an opaque document (e.g. a PDF)
 - `MovingImageRepresentation`: A resource representing a video
-- `ArchiveRepresentation`: A resource representing an archive file (e.g. ZIP)
+- `StillImageRepresentation`: A resource representing an image
+- `TextRepresentation`: A resource representing a text
```

Comment:
I see you ordered this alphabetically. Previously it was supposed to be (but wasn't really) ordered by importance/similarity, I think.  
E.g. it would make sense to me to group Document and Text representation; to have StillImage further up than Audio and Video; and especially to have 3D at the very bottom, as it's not implemented.  
But that's just a thought. (and I'm not saying the previous order was sensible...)